.github/actions/e2e/slack/notify/action.yaml

```
@@ -40,9 +40,9 @@ runs:
       if: ${{ job.status == 'success' }}
       with:
         url: ${{ inputs.url }}
-        message: ":white_check_mark: ${{ env.RUN_NAME }} tests succeeded (#${{ github.run_number }})"
+        message: ":white_check_mark: ${{ env.RUN_NAME }} https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
```

```suggestion
        message: ":white_check_mark: ${{ env.RUN_NAME }} https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}"
```

You're missing quotes here as well